code/modules/antagonists/eldritch_cult/eldritch_items.dm

```diff
@@ -180,3 +189,219 @@
+	hitsound = 'sound/weapons/bladeslice.ogg'
+	attack_verb_continuous = list("attacks", "slashes", "stabs", "slices", "tears", "lacerates", "rips", "dices", "rends")
+	attack_verb_simple = list("attack", "slash", "stab", "slice", "tear", "lacerate", "rip", "dice", "rend")
+	var/static/list/blacklisted_turfs = typecacheof(list(/turf/closed,/turf/open/space,/turf/open/lava))
+	var/drawing = FALSE
+	var/list/current_runes = list()
+	var/max_rune_amt = 3
+	var/datum/action/innate/rune_shatter/linked_action
+
+/obj/item/melee/rune_knife/Initialize()
+	. = ..()
+	linked_action = new(src)
+
+/obj/item/melee/rune_knife/pickup(mob/user)
+	. = ..()
+	linked_action.Grant(user, src)
+
+/obj/item/melee/rune_knife/dropped(mob/user, silent)
+	. = ..()
+	linked_action.Remove(user, src)
+
+/obj/item/melee/rune_knife/afterattack(atom/target, mob/user, proximity_flag, click_parameters)
+	. = ..()
+	if(!is_type_in_typecache(target,blacklisted_turfs) && !drawing && proximity_flag)
+		carve_rune(target,user,proximity_flag,click_parameters)
+
+/obj/item/melee/rune_knife/proc/carve_rune(atom/target, mob/user, proximity_flag, click_parameters)
+	var/obj/structure/trap/eldritch/elder = locate() in range(1,target)
+	if(elder)
+		to_chat(user,"<span class='notice'>You can't draw runes that clsoe to each other!</span>")
+		return
+
+	for(var/X in current_runes)
+		var/obj/structure/trap/eldritch/eldritch = X
+		if(QDELETED(eldritch) || !eldritch)
+			current_runes -= eldritch
+
+	if(current_runes.len == max_rune_amt)
+		to_chat(user,"<span class='notice'>The blade cannot support more runes!</span>")
+		return
+
+	var/list/pick_list = list()
+	for(var/E in subtypesof(/obj/structure/trap/eldritch))
+		var/obj/structure/trap/eldritch/eldritch = E
+		pick_list[initial(eldritch.name)] = eldritch
+
+	drawing = TRUE
+
+	var/type = pick_list[input(user,"Choose the rune","Rune") in pick_list]
+
+	to_chat(user,"<span class='notice'>You start drawing the rune...</span>")
+	if(!do_after(user,5 SECONDS,target = target))
+		drawing = FALSE
+		return
+
+	drawing = FALSE
+	var/obj/structure/trap/eldritch/eldritch = new type(target)
+	eldritch.set_owner(user)
+	current_runes += eldritch
+
+/datum/action/innate/rune_shatter
+	name = "Rune break"
+	desc = "Destroys all runes that were drawn by this blade."
+	background_icon_state = "bg_ecult"
+	button_icon_state = "rune_break"
+	icon_icon = 'icons/mob/actions/actions_ecult.dmi'
+	check_flags = AB_CHECK_CONSCIOUS
+	var/obj/item/melee/rune_knife/sword
+
+/datum/action/innate/rune_shatter/Grant(mob/user, obj/object)
+	sword = object
+	return ..()
+
+/datum/action/innate/rune_shatter/Activate()
+	for(var/X in sword.current_runes)
+		var/obj/structure/trap/eldritch/eldritch = X
+		if(!QDELETED(eldritch) && eldritch)
+			qdel(eldritch)
+
+/obj/item/eldritch_potion
+	name = "Brew of Day and Night"
+	desc = "You should never see this"
+	icon = 'icons/obj/eldritch.dmi'
+	var/status_effect
+
+/obj/item/eldritch_potion/attack_self(mob/user)
```

Comment:
people will try to use these by clicking on themselves with them; might want to add in support for that

also, are you meant to be able to drink this while your mouth is covered?

Reply:
good idea, and yes